requirements.in

```diff
@@ -0,0 +1,38 @@
+click==7.0
+dj-database-url==0.5.0
+django-autoslug==1.9.4
+django-click==2.1.0
+django-codemirror-widget==0.4.1
+django-countries==5.3.3
+django-date-extensions==3.0
+django-extensions<3
+django-formtools==2.1
+django-gulp-rev==0.2
+django-leaflet==0.24.0
+django-markdown-deux==1.0.5
+django-recaptcha==1.4.0
+django-storages[boto3]==1.7.1
+django-suit-redactor==0.0.4
+
+django-suit==0.2.26
+django-unused-media==0.1.13
+# Django suit breaks with Django 2.2
+django==2.0.12
```

Comment:
So this is the key one. Getting back onto a supported Django would be a good step. 

But yes, we can bump Django 2.0 to latest. 👍

Reply:
I saw some talk of people running django-suit from forks which have support for django 3 so there's a potential way to get django updated that way while replacing the functionality django-suit brings.

https://github.com/darklow/django-suit/issues/475#issuecomment-856635914